components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth/listener/OAuthApplicationMgtListener.java

```diff
@@ -64,4 +66,16 @@ public interface OAuthApplicationMgtListener {
      * @throws IdentityOAuthAdminException in case of failure.
      */
     void doPreRemoveOAuthApplicationData(String consumerKey) throws IdentityOAuthAdminException;
+
+    /**
+     * Pre-listener to handle the JWT token revocation.
+     *
+     * @param consumerKey consumer key.
+     * @param properties properties.
+     * @throws IdentityOAuthAdminException in case of failure.
+     */
+    default void doPostRevokeRegenerateOAuthSecret(String consumerKey, Properties properties)
```

Comment:
This should be doPostRegenerateOAuthSecret() right?

Revoking the token should be done by someone implementing this listener right?

Reply:
I prefer using doPostRegenerateOAuthSecret --> doPostRegenerateClientSecret :)

Reply:
@mefarazath fixed with e5008f6f293e8031c855ad4e5a6aaf736350952f